docs/requirements/stakeholder/index.rst

```diff
@@ -17,6 +17,739 @@
 Stakeholder Requirements
```

Comment:
Is Stakeholder still the right wording. I think we had that point already. Another thing would be to have a description not only a headline would be great

Reply:
I missed this discussion but would be open to change the wording here. Any suggestions?